From eb98fd30fc9e0336483d37c1fbb0b2363e09e932 Mon Sep 17 00:00:00 2001
From: Andy Lavr <andy.lavr@gmail.com>
Date: Mon, 24 Dec 2018 21:51:05 +0200
Subject: [PATCH] kernel: time: Fix posix-cpu-timers.c for PDS

Signed-off-by: Andy Lavr <andy.lavr@gmail.com>
---
 kernel/time/posix-cpu-timers.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/kernel/time/posix-cpu-timers.c b/kernel/time/posix-cpu-timers.c
index fb68143ab1de..7901b34d9ca2 100644
--- a/kernel/time/posix-cpu-timers.c
+++ b/kernel/time/posix-cpu-timers.c
@@ -921,8 +921,10 @@ static void check_process_timers(struct task_struct *tsk,
 	struct task_cputime cputime;
 	unsigned long soft;
 
+#ifndef CONFIG_SCHED_PDS
 	if (dl_task(tsk))
 		check_dl_overrun(tsk);
+#endif
 
 	/*
 	 * If cputimer is not running, then there are no active
-- 
2.18.1

